lib/split/helper.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 module Split
   module Helper
+    AB_OVERRIDE_PARAM = "ab_test"
```

Comment:
Named it this way because I thought this module was imported into user code, but it seems that doesn't happen directly, so maybe "AB_" could be stripped.